compiler/src/dmd/dinterpret.d

```diff
@@ -6704,7 +6704,7 @@ ThrownExceptionExp chainExceptions(ThrownExceptionExp oldest, ThrownExceptionExp
     // Little sanity check to make sure it's really a Throwable
     ClassReferenceExp boss = oldest.thrown;
     const next = 5;                         // index of Throwable.next
-    assert((*boss.value.elements)[next].type.ty == Tclass); // Throwable.next
+    assert((*boss.value.elements)[next].type.ty == Tpointer); // Throwable._nextInChainPtr
```

Comment:
for consistency, the comment in the line above too please :)

Reply:
oof, missed that...